internal/gribi/gribi.go

```diff
@@ -200,42 +200,41 @@ func (c *Client) AddNHG(t testing.TB, nhgIndex uint64, nhWeights map[uint64]uint
 // AddNH adds a NextHopEntry with a given index to an address within a given network instance.
 func (c *Client) AddNH(t testing.TB, nhIndex uint64, address, instance string, expectedResult fluent.ProgrammingResult, opts ...*NHOptions) {
 	t.Helper()
+	nh := fluent.NextHopEntry().
+		WithNetworkInstance(instance).
+		WithIndex(nhIndex)
 	switch address {
+	case "Vrf":
```

Comment:
Let's use `VRFOnly` to match the updated te-11.1 and te-11.2 tests (seems all three are about to be merged)

Reply:
Updated